test/C/src/federated/SimpleFederatedAuthenticated.lf

```diff
@@ -0,0 +1,23 @@
+/**
+ * This simple test checks if federate authentication works by adding `auth`
+ * target property.
+ */
+target C {
+    timeout: 2 secs,
+    build-type: RelWithDebInfo,
+    auth: true,  // FIXME: This will be updated to string such as HMAC.
```

Comment:
Same here. The policy is to not leave dangling `FIXME`s in `master`.

Reply:
Fixed it. I won't leave them next time!